Tests/MLXTests/MLXFastKernelTests.swift

```diff
@@ -70,4 +70,30 @@ class MLXFastKernelTests: XCTestCase {
         XCTAssertTrue(allClose(out[0], full([2, 2], values: 14.0484)).all().item())
         XCTAssertTrue(allClose(out[1], full([3, 2], values: -2)).all().item())
     }
+
+    func testFastSDPA() {
```

Comment:
Test case to make sure these can be called.  Without the fix this was able to reproduce the error.